client/app/bundles/course/video/submission/containers/DiscussionElements/Topic.jsx

```diff
@@ -60,7 +64,22 @@ function mapStateToProps(state, ownProps) {
   };
 }
 
-const TopicContainer = connect(mapStateToProps)(Topic);
+function mapDispatchToProps(dispatch) {
+  return {
+    onTimeStampClick: timestamp => (() => dispatch(updatePlayerProgress(timestamp, true))),
```

Comment:
What happens if for some reason, the timestamp is greater than the duration of the video? 

This probably won't happen, unless the underlying video is changed.

Reply:
It'll seek to the end of the video then.

It's defined in `reducers/video.js` on line 40.